crates/re_space_view/src/data_query_blueprint.rs

```diff
@@ -96,6 +97,64 @@ impl DataQueryBlueprint {
                 .join(&Self::RECURSIVE_OVERRIDES_PREFIX.into()),
         }
     }
+
+    pub fn remove_entity_expr(&self, ctx: &ViewerContext<'_>, expr: EntityPathExpr) {
+        let mut edited = false;
+
+        let mut inclusions: Vec<EntityPathExpr> = self
+            .expressions
+            .inclusions
+            .iter()
+            .filter(|exp| !exp.as_str().is_empty())
+            .map(|exp| EntityPathExpr::from(exp.as_str()))
+            .collect();
```

Comment:
nit: gives me deja vu. Maybe you want a utility to collect non-empty exclusions/inclusions